src/fix/main.c

```diff
@@ -528,6 +635,38 @@ static char const *mbcName(enum MbcType type)
 		return "HUC3";
 	case HUC1_RAM_BATTERY:
 		return "HUC1+RAM+BATTERY";
+	case TPP1:
+		return "TPP1";
+	case TPP1_RUMBLE:
+		return "TPP1+RUMBLE";
+	case TPP1_MULTIRUMBLE:
+		return "TPP1+MULTIRUMBLE";
+	case TPP1_MULTIRUMBLE_RUMBLE:
+		return "TPP1+MULTIRUMBLE+RUMBLE";
```

Comment:
The `help` documentation doesn't say `MULTIRUMBLE+RUMBLE`, just `MULTIRUMBLE`, since `RUMBLE` is implied.